hmc

```diff
@@ -192,6 +192,7 @@ function docker_build() {
     tagName=$(tag_name $project $branch)
     # hmi-api-gateway image from azure coming as hmi-api-gateway-web, to match and fix for local branch build
```

Comment:
is this comment correct? as we don't have a gateway built yet?

Reply:
Deleted the comment as it not relevant now